client/my-sites/subscribers/hooks/use-unsubscribe-modal.ts

```diff
@@ -29,7 +29,7 @@ const useUnsubscribeModal = (
 	const onConfirmModal = ( action: UnsubscribeActionType, subscriber?: Subscriber ) => {
 		if ( action === UnsubscribeActionType.Manage ) {
 			recordRemoveModal( true, 'manage_button_clicked' );
-			window.open( getEarnPaymentsPageUrl( selectedSiteSlug ), '_blank' );
+			navigate( `/earn/supporters/${ selectedSiteSlug ?? '' }` );
```

Comment:
@edanzer here we can link to individual payer's page, but I'm not sure if that's possible with how currently folks with multiple payments are linked, so might make sense only in future.

Linking to right place would make this flow much easier for publisher tho.